### PR TITLE
Add views and forms: Here is a PR for adding both views and forms to this Project. 

### PR DESCRIPTION
**Pull request by CrystallineButterfly**
Hello there, thank you for the review! 🦋

This PR covers the completion of the tasks at the 2 links below = 

Views: https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md ✅

Forms: https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md ✅

All the best- K 🦉

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,5 +4,9 @@ def index
     @posts = @user.recent_3_posts
   end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:id])
+    @comments = @post.comments.order('created_at')
+  end
```
Comment (GzimAsani) [inline app/controllers/posts_controller.rb:11]: - [x] Create current_user method on `application_controller` 
```ruby
# app/controllers/application_controller.rb
...

def current_user
  User.first
end
```

Comment (GzimAsani) [reply app/controllers/posts_controller.rb:11]: - [x] Please create the new and create actions on the post_controller

File: config/routes.rb [hunk 2]
```diff
@@ -3,4 +3,4 @@
   resources :users, only: [:index, :show] do
     resources :posts, on: :show, only: [:index, :show]
```
Comment (GzimAsani) [inline config/routes.rb:4]: - [x] There should be a route for creating new posts

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (GzimAsani) [reply app/controllers/posts_controller.rb:11]: - [x] In your views please add the respective form in the views/posts/new.html.erb file which accepts input for creating posts.

File: app/javascript/application.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
```
Comment (GzimAsani) [inline app/javascript/application.js:1]: - [x] All the previous requirements for `posts` such as creating the respective route, creating method and view, also apply for the creation of the comments

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @CrystallineButterfly       :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project, nice styling:+1:  
 
 ## Required Changes 

- [x] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @CrystallineButterfly , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CrystallineButterfly** [comment]
@GzimAsani Thank you for the helpful review, I have implemented the changes now! 🦋

**Comment by arslanbisharat** [comment]
Hi @CrystallineButterfly,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CrystallineButterfly** [comment]
Perfect! Thank you very much @arslanbisharat

**Pull request merged**